src/views/PopupView/InsertLabelNamesPopup/InsertLabelNamesPopup.scss

```diff
@@ -76,6 +76,24 @@
 
       color: white;
 
+      .ImageButton {
+        transition: transform 0.3s;
+
+        img {
+          filter: brightness(0) invert(1);
+          user-select: none;
+        }
+
+        &:hover {
+          background-color: transparent;
+        }
+
+        &:not(.disabled):hover {
+          filter: brightness(35%) sepia(100%) hue-rotate(172deg) saturate(2000%); // fallback if new css variables are not supported by browser
```

Comment:
ostatnio Ci powiedziałem że to jest fallback i to powinno działać xD ale chyba mi nie działało ostatnio xD 
to tylko tak informacyjnie :)